browser-interface/packages/shared/voiceChat/sagas.ts

```diff
@@ -89,6 +89,8 @@ function* handleConnectVoiceChatToRoom() {
           yield put(setVoiceChatError(err.toString()))
         }
       }
+    } else if (prevHandler) {
+      prevHandler.setMute(true)
```

Comment:
What is the idea about this? Can we add a comment with the expected behavior?

Reply:
well, since the prevHandler is not destroyed, just not used, we need to mute it in the meantime